### ISSUE TITLE
[BUG] Missing Accessibility Tab Indexing

### ISSUE THREAD

**Issue by compcon** [massif-press/compcon#1557]
<strong>Submitted At: </strong> 7/29/2021 0:56:35<br><br><strong>Browser: </strong> Firefox<br><strong>Platform: </strong> Desktop<br><br><strong>Issue Description: </strong><br> Buttons such as the homescreen selection (pilot, compendium, etc), or clicking on a specific character sheet in the pilot roster, are missing tab indices. When using only a keyboard (many people without screens, ie those using screen readers, do not use a mouse pointer to navigate), you cannot select these at all.<br><br><strong>Reproduction Steps:<br> </strong> 1. Press tab repeatedly to cycle through the buttons and links on-screen.
2. Several buttons will be skipped entirely.<br><br><strong>Expected Result:<br> </strong> I expect to be able to use the tab key to select between (and space to "click") all links and buttons for page navigation.<br><br><strong>Additional Context:<br> </strong> <br><br><br><strong>Stack Trace: </strong> No errors are relevant here, this is a design oversight.<br>

**Comment by Qazzquimby** [comment]
I'm working on this.
The issue is that `v-btn`s that have an `href` value use an `a` tag rather than a `button`. This happens regardless of if you set `tag="button"`, which sort of looks like a bug in vuetify, unless I'm missing something.

It can be fixed manually by adding `tabindex="0"` to the `v-btn` when needed, but it would be easy to get wrong.
We could also subclass `v-btn` to set `tabindex` automatically, and replace all `v-btn`s with the new component, but that sounds like overkill.

I'll do the manual way?

**Comment by msprijatelj** [comment]
@Qazzquimby I'll admit that I'm out of my element when it comes to adding accessibility options. Setting `tabindex="0"` seems like a lot of hardcoding that would be easy to slip up, especially if a new contributor is making changes. Replacing all `v-btn`s feels a bit better: It could possibly be resolved with a judicious "find and replace", and would be more friendly to contributors.

I found this entry on the Vuetify website for accessibility? I don't know if it'll help much with `v-btn` components but it seems to use the second approach of "subclassing `v-btn`: https://vuetifyjs.com/en/features/accessibility/#activator-slots

**Comment by Qazzquimby** [comment]
Note that this issue will recur unless everyone tests for keyboard accessibility before submitting work.

**Comment by msprijatelj** [comment]
@Qazzquimby would you mind adding documentation on what you did in CONTRIBUTORS.md for reference? It would greatly help newcomers maintain accessibility.
